src/VirtoCommerce.PricingModule.Web/Scripts/blades/pricelist-item-list.js

```diff
@@ -1,6 +1,7 @@
 angular.module('virtoCommerce.pricingModule')
     .controller('virtoCommerce.pricingModule.pricelistItemListController', ['$scope', 'virtoCommerce.pricingModule.prices', '$filter', 'platformWebApp.bladeNavigationService', 'uiGridConstants', 'platformWebApp.uiGridHelper', 'platformWebApp.bladeUtils', 'platformWebApp.dialogService', function ($scope, prices, $filter, bladeNavigationService, uiGridConstants, uiGridHelper, bladeUtils, dialogService) {
         $scope.uiGridConstants = uiGridConstants;
+        $scope.defaultProductName = "Product with the given Id does not exist";
```

Comment:
Could you add localization for this row?
Also could you rename it to something like e.g. `noProductRowName`